### PR TITLE
add generated federation docs for 1.10

### PR DESCRIPTION
**Pull request by Bradamant3**
/assign @zacharysarah 
/assign @steveperry-53 

/hold

### CODE REVIEW

**Comment by zacharysarah** [comment]
@Bradamant3 It looks like Travis is failing on the removal of a `title:` field. Is that an intentional removal, or is it solely copypasta of the script output?

**Comment by steveperry-53** [comment]
@Bradamant3, As @zacharysarah points out, the titles are missing. Apparently the generation script doesn't do titles, and we will have to paste them in manually.

**Comment by steveperry-53** [comment]
/approve
/lgtm
@Bradamant3 I think it's OK to remove the hold whenever you're ready.

**Comment by zacharysarah** [comment]
@Bradamant3 Bump

**Comment by zacharysarah** [comment]
@Bradamant3 Bump. Looks like there are merge conflicts to resolve.

File: docs/reference/generated/federation/extensions/v1beta1/definitions.html [hunk 1]
```diff
@@ -2591,7 +2591,7 @@ <h3 id="_v1_envfromsource">v1.EnvFromSource</h3>
 <tbody>
 <tr>
 <td class="tableblock halign-left valign-top"><p class="tableblock">prefix</p></td>
-<td class="tableblock halign-left valign-top"><p class="tableblock">An optional identifier to prepend to each key in the ConfigMap. Must be a C_IDENTIFIER.</p></td>
+<td class="tableblock halign-left valign-top"><p class="tableblock">An optional identifer to prepend to each key in the ConfigMap. Must be a C_IDENTIFIER.</p></td>
```
Comment (mistyhacks) [inline docs/reference/generated/federation/extensions/v1beta1/definitions.html]: Nit: Introduced a typo

**Comment by Bradamant3** [comment]
/hold cancel
I think we can live with the typo for now. I'll go in and fix the source in the meantime, though. And let me know if you disagree, @zacharysarah 😄

**Comment by Bradamant3** [comment]
o right, all the new commits.

/assign @zacharysarah

**Pull request merged**